docs/user/explanation/documentation-structure.rst

```diff
@@ -0,0 +1,43 @@
+How to structure your documentation
+===================================
+
+Structuring your documentation is an important part of the authoring process.
+It's important to think about the reader of the documentation,
+since they are the primary user of documentation.
+
+One of the largest benefits of a good structure is that it removes questions that keep authors from writing documentation.
+Starting with a blank page is often the hardest part of documentation,
+so anything we can do to remove this problem is a win.
+
+Recommendations for process
+---------------------------
```

Comment:
```suggestion
Recommendations a structure
---------------------------

Good news!
You don't have to invent all of the structure yourself,
since a lot of experience-based work has been done to come up with a universal documentation structure.
```

Maybe we should also change our example projects to have a Diátaxis skeleton? :)